mongo/init-mongo.js

```diff
@@ -21,6 +21,16 @@ if (db.counters.countDocuments({ _id: "architectureStoreCounter" }) === 1) {
     print("architectureStoreCounter already exists, no initialization needed");
 }
 
+if (db.counters.countDocuments({ _id: "flowStoreCounter" }) === 1) {
+    db.counters.insertOne({
+        _id: "flowStoreCounter",
+        sequence_value: 1
```

Comment:
I think if I was to play around a bit more with the swagger api, I probably will not be able to add documents because this init mongo script adds more documents then the initial sequence number.